src/app/globals.css

```
@@ -55,6 +55,22 @@
 }
 
+@font-face {
+  font-family: 'NeoDunggeunmo';
+  src: url('/fonts/neodgm.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Roboto';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)